### DISCUSSION TITLE
Support sessionStorage in auth.Strategies options

### DISCUSSION THREAD

**Discussion by mtskf** [nuxt-community/auth-module#1626] [Ideas & Feature Requests]
### Is your feature request related to a problem? Please describe.
Local storage was never meant to be used as secure storage; the token data in the localStorage can be easily extracted even after closing the app and reused. Thus, sessionStorage is preferable.

### Describe the solution you'd like to see
It would be great if there's an "session" option in the auth.strategies options, which uses window.sessionStorage to store the JWT instead of window.localStorage.

Thanks so much for providing the awesome module, I really love it! 👍
I'd appreciate if you could support the feature! It would be really life saving. 😊

**Comment by Intevel** [comment]
Hey @mtskf, I added this in #1441

**Comment by mjrussell91** [comment]
Agreed. The requirements for auth on my project are such that Session Storage is preferable to Local Storage/Cookies. Plus I feel that it's good security practice in general to have this option.

This feature has my support.
